Documentation/cmdref/cilium-agent.md

```diff
@@ -202,7 +202,7 @@ cilium-agent [flags]
       --hubble-export-file-compress                               Compress rotated Hubble export files.
       --hubble-export-file-max-backups int                        Number of rotated Hubble export files to keep. (default 5)
       --hubble-export-file-max-size-mb int                        Size in MB at which to rotate Hubble export file. (default 10)
-      --hubble-export-file-path string                            Filepath to write Hubble events to.
+      --hubble-export-file-path stdout                            Filepath to write Hubble events to. By specifying stdout the flows are logged instead of written to a rotated file.
```

Comment:
it's strange that the type of this has now changed from `string`.

Reply:
Yeah...that is quite weird actually.

Reply:
Is there anything I can/should do about this?
Btw. my commit is now rebased and all tests should work.

Reply:
This file is auto-generated so if there is something we want to change about this, we'd have to track it down and fix the generator. There is no action on your part :)